hypervisor/boot/guest/vboot_info.c

```diff
@@ -225,7 +225,10 @@ static int32_t init_vm_sw_load(struct acrn_vm *vm, const struct acrn_multiboot_i
 	}
 
 	if (ret == 0) {
-		init_vm_bootargs_info(vm, mbi);
+		/* Currently VM bootargs only support Linux guest */
+		if (vm->sw.kernel_type == KERNEL_BZIMAGE) {
```

Comment:
You implicitly mean KERNEL_BZIMAGE is only used by Linux.  Please make the comment more clear on this.

Reply:
Updated the comments to "Currently VM bootargs only support Linux kernel bzImage format".